test/files/ala002/pass-ala002-helmert_curve.ifc

```diff
@@ -0,0 +1,156 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [CoordinationView]'), '2;1');
+FILE_NAME ('example-single-transition-curve.ifc', '2022-02-22T22:00:00', ('IFC Rail Technical Services'), ('buildingSMART International RailwayRoom'), 'RDF Ltd. (https://rdf.bg/)', '2019.1.1995.0', '');
+FILE_SCHEMA (('IFC4X3_ADD2'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('1FNFy8AJeHwwz7wDZHIYIu', #3, 'IFC Rail Referencedata', 'Rail alignment sample', $, 'Longname', 'Design', $, #9);
+#2 = IFCAPPLICATION(#5, '2019.1.1995.0', 'IFCRAIL', 'https://rdf.bg/');
+#3 = IFCOWNERHISTORY(#6, #2, $, .ADDED., 1320688800, $, $, 1320688800);
+#4 = IFCPERSON('TS', 'IFC Rail Technical Services', $, $, $, $, $, $);
+#5 = IFCORGANIZATION('bsI RWR', 'buildingSmart International Railway Room', $, $, $);
+#6 = IFCPERSONANDORGANIZATION(#4, #5, $);
+#7 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#8 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#9 = IFCUNITASSIGNMENT((#7, #8));
+#10 = IFCCARTESIANPOINT((0., 0., 0.));
+#11 = IFCDIRECTION((0., 0., 1.));
+#12 = IFCDIRECTION((1., 0., 0.));
+#13 = IFCAXIS2PLACEMENT3D(#10, #11, #12);
+#14 = IFCLOCALPLACEMENT($, #13);
+#15 = IFCRAILWAY('1FNFy9AJeHwuVmwDZHIYIu', #3, 'optional Railway Name', 'optional Railway Description', $, #14, $, 'Longname for Railway track geometry (alignment)', $, $);
+#16 = IFCDIRECTION((0., 1.));
+#17 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'MODEL', 3, 1.E-5, #13, #16);
+#18 = IFCRELAGGREGATES('1FNFyAAJeHwu6TwDZHIYIu', #3, 'Documentation Container', 'Container for IfcProject, collects relevant products', #1, (#15));
+#19 = IFCRELCONTAINEDINSPATIALSTRUCTURE('1FNFyBAJeHwvmywDZHIYIu', #3, 'IfcProducts contained in myIfcRailway', 'Association between IfcProducts and IfcRailway', (#20), #15);
+#20 = IFCALIGNMENT('1FNFyCAJeHwxedwDZHIYIu', #3, 'Spor', 'optional Railway Description', $, #153, #156, $);
+#21 = IFCALIGNMENTHORIZONTAL('1FNFyDAJeHwv87wDZHIYIu', $, $, $, $, #94, $);
+#23 = IFCRELNESTS('3BJTAQrjCHwvVKbERtTLTf', $, $, $, #20, (#21, #41, #61));
+#28 = IFCCARTESIANPOINT((0., 0.));
+#29 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #28, 0., 300., 0., 100., $, .HELMERTCURVE.);
```

Comment:
It would be nice to have a file that fails for specifically for helmertcurves as well. 
Can't this be as simple as taking a file that now passes and change the predefined types of one segment to `HELMERTCURVE`. If I'm correct, it should fail then. For instance, the file `test/files/ala002/pass-ala002-business-logic_only.ifc`

Reply:
Added a `fail` file for Helmert Curve